src/components/productDetail.js

```diff
@@ -11,28 +12,28 @@ function ProductDetail({ productInformation }) {
     ? "Agregar al carrito"
     : "En carrito";
   return (
-    <div className="d-flex">
+    <div className="product detail d-flex">
       <div>
-        <img src={productInformation.imageUrl} alt="Product" />
+        <img src={productInformation.imageUrl} alt="Product" className="imgb"/>
       </div>
       <div>
         <div>
-          <p>Categorias</p>
+          <p id="titulo-categorias">Categorias</p>
           <div>
-            <p>Categoria 1</p>
-            <p>Categoria 2</p>
+            <p id="categories">Categoria 1</p>
+            <p id="categories">Categoria 2</p>
           </div>
         </div>
-        <h2>{productInformation.name}</h2>
-        <div>
-          <p>Star section</p>
+        <h2 className="titulo-producto">{productInformation.name}</h2>
+        <div className="star-section">
+          <p>★★★★☆</p>
```

Comment:
Is this value changing when the ratingAverage is different than 4?

Reply:
That rating was coded for the vendor section, not for the product rating. The stars are simulating the rating

Reply:
Could you implement the css to be able to update the stars according to the rating? On this component you have available the `productInformation.ratingAverage` variable to know how many stars you have to show.